src/components/App.js

```diff
@@ -1,11 +1,34 @@
+import React, { Component } from 'react';
```

Comment:
### Optional
- It would be better to update the live link so we have the recent update, now is a good time to do that right?